src/averageAges.js

```diff
@@ -37,7 +50,25 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mothersNames = people
+    .filter(({ mother }) => mother)
+    .reduce((acc, person) => {
+      return acc.concat(person.mother);
+    }, []);
```

Comment:
You don't really need this. You can check if a woman is a mother using `some` when you are filtering them on lines 59-63